packages/postgres/scripts/ensure-db-exists.sh

```diff
@@ -1,5 +1,27 @@
 #!/bin/sh
 
+# wait until postgres inside the container accepts connections
+wait_for_postgres() {
+  ATTEMPTS=0
+  MAX_ATTEMPTS="${PG_WAIT_ATTEMPTS:-60}"
+
+  while [ "$ATTEMPTS" -lt "$MAX_ATTEMPTS" ]; do
+    if docker exec boxel-pg pg_isready -U postgres >/dev/null 2>&1; then
+      return 0
+    fi
+
+    ATTEMPTS=$((ATTEMPTS + 1))
+    sleep 1
+  done
+
+  echo "Timed out waiting for Postgres to become ready after $MAX_ATTEMPTS seconds" >&2
+  return 1
+}
+
+if ! wait_for_postgres; then
+  exit 1
+fi
+
```

Comment:
I’m confused why this would be needed as other packages don’t have anything similar but without it I was seeing the job fail like [this](https://github.com/cardstack/boxel/actions/runs/18774342477/job/53565411587):

```
psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
```

Maybe other things were waiting in a different way?